### ISSUE TITLE
[Question] AuthenticateAsync (Google) always false after auth flow return 

### ISSUE THREAD

**Issue by Cpousson** [xamarin/Essentials#1494]
I have the sample Authcontroller.cs and startup.cs for the WebAuthenticator setup using the Authentication.Google library. When the controller initially challenges and starts the flow, the google login page displays and I login.  It is succesfully redirected back to my controller. However, it checks auth again:

`var auth = await Request.HttpContext.AuthenticateAsync(scheme);`

which I then expect to now succeed and move to stripping the token etc. but it is always false. Am I missing a step?

**Comment by mattleibow** [comment]
@Redth, does this sound like anything you have seen? It sounds like this might be some configuration issue with the Google side as we are using the sample code right now. And, I just tested several things today with Google login and all was working.

**Comment by Redth** [comment]
Yeah I haven't seen this myself, it sounds like you need to figure out why that's happening on the asp.net side of things.  Perhaps the user has not allowed scopes/permissions requested? Not sure without more context/info.

**Comment by dunwan** [comment]
What fixed the false response for me was changing from;
`var result = await HttpContext.AuthenticateAsync(CookieAuthenticationDefaults.AuthenticationScheme);`
to
`var result = await HttpContext.AuthenticateAsync(GoogleDefaults.AuthenticationScheme);`
